src/store/reducers/cohortBuilder.js

```diff
@@ -55,6 +73,8 @@ export default (state = initialState, action) => {
       return {
         ...state,
         ...action.payload,
+        dirty: false,
+        areSqonsEmpty: false,
       };
     case VIRTUAL_STUDY_SAVE_FAILURE:
```

Comment:
@jberube Should it be the same as `default` clause; does this mean if there is a save error the state will lose all values?

Reply:
No, it won't?